src/MCPServer/lib/assets/workflow.json

```diff
@@ -7210,6 +7364,25 @@
         "sv": "Skapa SIP från överföringspaket"
       }
     },
+    "7e81f94e-6441-4430-a12d-76df09181b66": {
+      "config": {
+        "@manager": "linkTaskManagerChoice",
+        "@model": "MicroServiceChainChoice",
+        "chain_choices": [
+          "97be337c-ff27-4869-bf63-ef1abc9df15d",
+          "77355172-b437-4324-9dcc-e2607ad27cb1"
+        ]
+      },
+      "description": {
+        "en": "Do you want to scan for viruses in directories?"
+      },
+      "exit_codes": {},
+      "fallback_job_status": "Failed",
+      "fallback_link_id": null,
+      "group": {
+        "en": "Scan for viruses"
```

Comment:
Should we include the rest of existing translations (e.g. `fr`, `ja`, etc) for the `Scan for viruses` group here?